docs/API.md

```diff
@@ -81,7 +81,7 @@
     it is up to you to terminate the process; you **must** perform only synchronous operations at this point.
     See [Extreme mode explained](extreme.md) for more detail.
   * `enabled` (boolean): enables logging. Default: `true`
-  * `browser` (Object): browser only, may have `asObject` and `write` keys, see [Pino in the Browser](../readme.md#browser)
+  * `browser` (Object): browser only, may have `asObject` and `write` keys, see [Pino in the Browser](https://github.com/pinojs/pino#browser)
```

Comment:
I think the actual fix would be `(/README.md#browser)`.

Reply:
right I'm stupid I tested readme.md lowercase and now it's upper